cranelift/codegen/src/isa/riscv64/inst.isle

```diff
@@ -2047,13 +2040,6 @@
       (a2 Reg (alu_rr_imm12 (AluOPRRI.Addi) a (imm12_const 1))))
     (gen_select_reg (IntCC.SignedLessThan) r (zero_reg) a2 r)))
 
-(decl gen_trapff (FloatCC Reg Reg Type TrapCode) InstOutput)
-(rule
-  (gen_trapff cc a b ty trap_code)
-  (let
-    ((tmp WritableReg (temp_writable_reg $I64)))
-    (side_effect (SideEffectNoResult.Inst (MInst.TrapFf cc a b ty tmp trap_code)))))
-
```

Comment:
Was this change supposed to be in this PR? It looks like it's a part of #5515 that leaked into here.

Reply:
Thanks for catching that! Originally I was trying to remove `lower_br_fcmp` in this PR, but started feeling like it was becoming a bit unwieldy.